account_statement_import_sheet_file/readme/CONFIGURE.md

```diff
@@ -1,6 +1,35 @@
-To create TXT/CSV/XLSX statement sheet columns mapping:
+To configure this module, you need to:
+1. Install the module account_statement_import_sheet_file.
```

Comment:
Well, I think there is no need to state that we need to install it, isn't it? :wink: